lib/fluent/plugin/prometheus.rb

```diff
@@ -192,5 +194,33 @@ def instrument(record, expander, placeholders)
         end
       end
     end
+
+    class Histogram < Metric
+      def initialize(element, registry, labels)
+        super
+        if @key.nil?
+          raise ConfigError, "histogram metric requires 'key' option"
+        end
+
+        begin
+          if element['buckets']
+            buckets = element['buckets'].split(/,/).map do |e|
```

Comment:
The split makes space prefixed numbers such as " 0.10", which results in 0 by `to_i`. You need to call `map(&:strip)` after split.